helm/idol-nifi/resources/flow-basic-idol.json

```diff
@@ -1,3685 +1,3982 @@
+                                        "prioritizers": [],
+                                        "bends": [],
+                                        "loadBalanceStrategy": "DO_NOT_LOAD_BALANCE",
+                                        "partitioningAttribute": "",
+                                        "loadBalanceCompression": "DO_NOT_COMPRESS",
+                                        "componentType": "CONNECTION",
+                                        "groupIdentifier": "95b7bc81-a0f0-30a4-ac56-e4933a396fd1"
+                                    },
+                                    {
+                                        "identifier": "b1d85447-d730-3c42-8057-ed71a95cd823",
+                                        "instanceIdentifier": "c3aaed67-33d5-39d6-7bca-0b381ba8e6b6",
+                                        "name": "",
+                                        "source": {
+                                            "id": "77d580a4-7b15-37d7-8c36-96358b87e227",
+                                            "type": "PROCESSOR",
+                                            "groupId": "95b7bc81-a0f0-30a4-ac56-e4933a396fd1",
+                                            "name": "GetFileSystem",
+                                            "comments": "",
+                                            "instanceIdentifier": "134924eb-eca6-3eaf-18ba-12bf7a0050d5"
+                                        },
+                                        "destination": {
+                                            "id": "78a3e3d8-c2ae-3053-800e-23b3a3963481",
+                                            "type": "FUNNEL",
+                                            "groupId": "95b7bc81-a0f0-30a4-ac56-e4933a396fd1",
+                                            "name": "Funnel",
+                                            "comments": "",
+                                            "instanceIdentifier": "d6882542-703b-369c-411a-fd58b5f48863"
+                                        },
+                                        "labelIndex": 1,
+                                        "zIndex": 0,
+                                        "selectedRelationships": [
+                                            "failed identifiers"
+                                        ],
+                                        "backPressureObjectThreshold": 10000,
+                                        "backPressureDataSizeThreshold": "1 GB",
+                                        "flowFileExpiration": "0 sec",
+                                        "prioritizers": [],
+                                        "bends": [],
+                                        "loadBalanceStrategy": "DO_NOT_LOAD_BALANCE",
+                                        "partitioningAttribute": "",
+                                        "loadBalanceCompression": "DO_NOT_COMPRESS",
+                         
```

Comment:
```suggestion
                                "label": "To add a connector group: \n- Create a process group with the desired connector router and relations.\n- Add a Get<connector> processor which, upon success, routes to the Synchronize Output output port.\n- In the ConnectorGroupRouter processor in the level above, add a new property for the connector group\n  (e.g. WebConnectorGroup with value Web) and create a connection to the new connector group\n  process group.\n- Make sure all failures are routed to the UnregisterDocument processor.",
                                "zIndex": 0,
```

Reply:
I'm not sure GH has rendered that suggestion correctly - intention is to update the "To add a connector group" instructions to generalize the Get processor being added.